testing/ton-lite-client/APKBUILD

```diff
@@ -0,0 +1,45 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+
+pkgname=ton-lite-client
+pkgver=0.0.1
+pkgrel=0
+pkgdesc="Telegram Open Network (TON) lite client"
+url="https://test.ton.org"
+arch="all !aarch64"
+license="LGPL-2.0-or-later"
+options="!check" # No test suite from upstream
+makedepends="cmake gperf linux-headers openssl-dev zlib-dev"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.xz::https://test.ton.org/ton-test-liteclient-full.tar.xz
+	https://test.ton.org/ton-lite-client-test1.config.json
+	"
+builddir="$srcdir/${pkgname##ton-}"
+
+build() {
+	mkdir build
+	cd build
+	cmake -DCMAKE_INSTALL_PREFIX=/usr --build .. 
+}
+
+package() {
+	cd $builddir/build
+	make DESTDIR=$pkgdir install
+
+	mkdir $pkgdir/usr/bin
+	rm -rf CMakeFiles
+	local i
+	for i in $(find . -executable -type f); do
+		cp $i $pkgdir/usr/bin/
+	done
```

Comment:
Why doesn't the CMake build system do this?

Reply:
Sorry, I have no answer. Must be for the very early stage of the project. Plane to give a closer look at CMake build system.